src/osemosys/schemas/base.py

```diff
@@ -69,11 +69,42 @@ class RegionYearData(BaseModel):
         Dict[str,Dict[int,float]]
     ]
 
+class StringYearData(BaseModel):
```

Comment:
lets discuss how we can make this more generic, maybe with `Any` or some generic set of `float,int,str,DateTime`. We can write some validation to set what data has been input